src/components/EmojiStaticKeyboard.tsx

```diff
@@ -47,6 +47,19 @@ export const EmojiStaticKeyboard = () => {
     }).start();
   }, [activeCategoryIndex, scrollNav]);
 
+  const renderList = React.useCallback(() => {
```

Comment:
replace this useCallback with useMemo:
-this function is called on every rerender, so it must calculate new array
-this function always returns different array(reference)